utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -432,6 +432,10 @@ int main(int argc, char **argv)
     if(testCase == 28)
```

Comment:
we should use ENUM for test case instead of hardcoded values for better code readability

Reply:
Addressed with #499